reinforcement_learning/rl_hvac_coach_energyplus/rl_hvac_coach_energyplus.ipynb

```diff
@@ -489,9 +489,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "predictor = estimator.deploy(initial_instance_count=1,\n",
```

Comment:
Any reason for this change? Does MXNet 1.8.0 compatible with what exported from Ray

Reply:
Good question, I did not see the hosting succeed with the generic `estimator.deploy` function, which is why I replaced that with a `MXNetModel` instead. The `1.8.0` version is the latest and the notebook didn't throw any errors, so I'm assuming its backwards compatible with the MXNet version (1.3.0) of the sagemaker-rl coach image for hosting the model.

Reply:
Others LGTM